res/openvpn-gui-res-ru.rc

```diff
@@ -535,34 +535,34 @@ OpenVPN, возможно, не установлен."
     IDS_ERR_IMPORT_FAILED "Ошибка импорта файла. Путь %ls не может быть создан.\n\n\
 Убедитесь, что у вас есть необходимые привилегии для данного действия."
     IDS_NFO_IMPORT_SUCCESS "Импорт файла успешно завершен."
-    IDS_NFO_IMPORT_OVERWRITE "A config named ""%ls"" already exists. Do you want to replace it?"
-    IDS_NFO_IMPORT_SOURCE "Do you want to import the profile <%ls>?"
-    IDS_ERR_IMPORT_SOURCE "Cannot import file <%ls> as it is already in the global or local config directory"
-    IDS_ERR_IMPORT_ACCESS "Cannot import <%ls> as it is missing or not readable"
+    IDS_NFO_IMPORT_OVERWRITE "Файл конфигурации с именем ""%ls"" уже существует. Вы хотите заменить его?"
+    IDS_NFO_IMPORT_SOURCE "Вы хотите импортировать профиль <%ls>?"
+    IDS_ERR_IMPORT_SOURCE "Нельзя импортировать файл <%ls>, поскольку он уже в папке глобальных или локальных настроек"
+    IDS_ERR_IMPORT_ACCESS "Нельзя импортировать файл <%ls>, поскольку он отсутствует или не может быть прочитан"
 
     /* save/delete password */
     IDS_NFO_DELETE_PASS "Подтвердите удаление сохраненных паролей для ""%ls"""
 
     /* Token password related */
-    IDS_NFO_TOKEN_PASSWORD_CAPTION "OpenVPN – Token Password"
-    IDS_NFO_TOKEN_PASSWORD_REQUEST "Input Password/PIN for Token '%hs'"
+    IDS_NFO_TOKEN_PASSWORD_CAPTION "OpenVPN – токен-пароль"
+    IDS_NFO_TOKEN_PASSWORD_REQUEST "Введитте пароль/PIN для токена '%hs'"
 
     IDS_NFO_AUTH_PASS_RETRY "Неправильное имя пользователя или пароль"
     IDS_NFO_KEY_PASS_RETRY  "Неправильный пароль"
-    IDS_ERR_INVALID_PASSWORD_INPUT "Invalid character in password"
-    IDS_ERR_INVALID_USERNAME_INPUT "Invalid character in username"
+    IDS_ERR_INVALID_PASSWORD_INPUT "Недопустимый символ в пароле"
+    IDS_ERR_INVALID_USERNAME_INPUT "Недопустимый символ в имени пользователя"
     IDS_NFO_AUTO_CONNECT    "Автоматическое подключение через %u сек..."
-    IDS_NFO_CLICK_HERE_TO_START "OpenVPN GUI is already running. Right click on the tray icon to start."
-    IDS_NFO_BYTECOUNT "Bytes in: %ls  out: %ls"
+    IDS_NFO_CLICK_HERE_TO_START "OpenVPN GUI уже запущен. Нажмите правой кнопкой на иконке в трее чтобы развернуть."
+    IDS_NFO_BYTECOUNT "Входящие байты: %ls  Исходящие байты: %ls"
 
     /* AS profile import */
-    IDS_ERR_URL_IMPORT_PROFILE "Error fetching profile from URL: [%d] %ls"
+    IDS_ERR_URL_IMPORT_PROFILE "Ошибка получения профиля по ссылке: [%d] %ls"
 
-    IDS_ERR_NO_PKCS11 "No certificates found. If you have a token insert it and press retry."
-    IDS_ERR_SELECT_PKCS11 "No certificates selected."
+    IDS_ERR_NO_PKCS11 "Сертификат не найден. Если у вас есть токен, введите его и нажмите повторить"
+    IDS_ERR_SELECT_PKCS11 "Сертификаты не выбраны."
     /* column headers for pkcs11 certificate list */
-    IDS_CERT_DISPLAYNAME "Issued to"
-    IDS_CERT_ISSUER "Issued by"
-    IDS_CERT_NOTAFTER "Valid until"
+    IDS_CERT_DISPLAYNAME "Выдано для:"
```

Comment:
We don't have colon in English version, so let's be consistent and not add it here.